Shoko.Server/Plex/PlexHelper.cs

```diff
@@ -158,26 +158,23 @@ private Connection ConnectionCache
 
     private Dictionary<string, string> AuthenticationHeaders => new() { { "X-Plex-Token", GetPlexToken() } };
 
+    private DateTime? _lastAuthenticated = null;
+
     public bool IsAuthenticated
     {
         get
         {
-            if (isAuthenticated is true)
+            if (_lastAuthenticated is not null && DateTime.Now - _lastAuthenticated < TimeSpan.FromMinutes(30))
             {
-                return isAuthenticated.GetValueOrDefault(false);
-            }
-
-            // If key is not null, then we are trying to login. Plex token check should be skipped.
-            if (_key == null && string.IsNullOrEmpty(_user?.PlexToken))
-            {
-                return false;
+                return true;
             }
 
             try
             {
                 isAuthenticated = RequestAsync("https://plex.tv/users/account.json", HttpMethod.Get,
                         AuthenticationHeaders).ConfigureAwait(false)
                     .GetAwaiter().GetResult().status == HttpStatusCode.OK;
+                _lastAuthenticated = DateTime.Now;
```

Comment:
```suggestion
                _lastAuthenticated = isAuthenticated ? DateTime.Now : null;
```

This will make it so a failed login doesn't false-trigger to a true.